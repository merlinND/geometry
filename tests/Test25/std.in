C myCircle 0 0 1
C mySecondCircle 1 1 42
OA myAgregate myCircle mySecondCircle
MOVE myAgregate 1 1
LIST
EXIT
