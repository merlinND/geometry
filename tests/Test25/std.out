OK
OK
OK
OK
OA myAgregate myCircle mySecondCircle
C myCircle 1 1 1
C mySecondCircle 2 2 42
